### PR TITLE
APIS - API Release State

### PR DESCRIPTION
**Pull request by larshp**
https://github.com/abapGit-tests/APIS

tested on 757, file format same as on SAP Business Technology Platform (BTP) ABAP Environment

https://github.com/abapGit/docs.abapgit.org/pull/159

### CODE REVIEW

File: src/objects/zcl_abapgit_object_apis.clas.abap [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+    METHODS initialize.
+
+ENDCLASS.
+
+
+
+CLASS ZCL_ABAPGIT_OBJECT_APIS IMPLEMENTATION.
+
+
+  METHOD constructor.
```
Comment (mbtools) [inline src/objects/zcl_abapgit_object_apis.clas.abap:30]: https://github.com/abaplint/abaplint/issues/1417 😉

Comment (larshp) [reply src/objects/zcl_abapgit_object_apis.clas.abap:31]: yea, work in progress, but will add the rule sometime

**Review by mbtools — APPROVED** [review]
👍

**Comment by larshp** [comment]
going fast, can also leave these for a few days for more to possibly review

but anyhow, wont matter much, as its new

**Pull request merged**